### DISCUSSION TITLE
Can't get started

### DISCUSSION THREAD

**Discussion by nickworks** [filamentphp/filament#12657] [Help] [answered]
### Package

Panel builder

### Package Version

v3.4.12

### How can we help you?

I'm trying to follow along with the "Getting Started" here: https://filamentphp.com/docs/3.x/panels/getting-started

However, when I get to the step "Introducing resources" ( `php artisan make:filament-resource Patient` )  I get the following error: `Call to a member function getResourceDirectories() on null`

Anyone know what I'm doing wrong here?? Thanks in advance!

Laravel: 11.6.0
PHP: 8.3.4
Composer: 2.7.2
Filament: 3.2.73
Livewire: 3.4.12

**Comment by leandrocfe** [comment]
Is it a fresh filament install?

**Reply by nickworks** [reply, marked as answer]
Yes it is. Here is my history:
```
 1758  laravel new test-filament
 1759  ls -la
 1760  cd test-filament/
 1761  ls -la
 1762  sdo vim .env
 1763  sudo mvim .env
 1764  php artisan migrate
 1765  php artisan serve
 1766  composer require filament/filament:"^3.2" -W
 1767  php artisan filament:install
 1768  php artisan make:model Owner -m
 1769  php artisan make:model Patient -m
 1770  php artisan make:model Treatment -m
 1771  code .
 1772  php artisan migrate
 1773  php artisan make:filament-resource Patient
```
At 1771 I added the model relationships and edited the migrations.

I almost submitted this, when I realized I should go back to double-check the [Installation](https://filamentphp.com/docs/3.x/panels/installation) page against my history above. Lo and behold, it looks like I missed the `--panels` flag at 1767 above.

The problem was solved by running `php artisan filament:install --panels`

Thanks for your help!

**Comment by msulaimanmisri** [comment]
I've got the same situation. After follow your solution, it is work now. But I think they should include the `php artisan filament:install --panels` at the `Getting Started` as this page is demoing how to build a simple application using Filament
